dataset_interfaces/interface.py

```diff
@@ -132,7 +140,7 @@ def action_iter(self) -> Iterator[TestAction]:
                 yield SendAndRegisterAction(msg, is_question=is_q)
             else:
                 yield SendMessageAction(msg, is_question=is_q)
-            if any(wait.values()):
+            if len(wait) > 0:
```

Comment:
Is this correct? The create_wait function will return by default this:  `{"tokens": None, "time": None, "percentage_finished": None}`

Reply:
Oh, I missed that `if not None` at the end of  line 74